frontend_build/src/parse_bundle_plugin.js

```diff
@@ -15,6 +15,7 @@ var extract$trs = require('./extract_$trs');
 var merge = require('webpack-merge');
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
 var WebpackRTLPlugin = require('webpack-rtl-plugin');
+var LodashModuleReplacementPlugin = require('lodash-webpack-plugin');
```

Comment:
Wondering what difference this makes to the final build, if we have already been careful about all of our lodash imports?

Reply:
This decreased sizes from 100-200kb to under 50kb. Its documentation says "it complements babel-plugin-lodash by shrinking its cherry-picked builds even further"!

Reply:
Wow, do we have an idea of what this is actually doing under the hood? Just wondering what the price is for these huge savings!

Reply:
I think it automatically removes the list of features here. https://github.com/lodash/lodash-webpack-plugin
We choose to keep features we need. I am not sure which features are used and which are not in kolibri though.